### ISSUE TITLE
Recurrent myocarditis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6417]
**Preferred term label:**
Recurrent myocarditis

**Synonyms**
Recurrent inflammation of the heart muscle

**Definition (free text, please give PubMed ID)**
Recurrent inflammation of the myocardium.
DOI: https://doi.org/10.1016/j.cardfail.2018.07.327

**Parent term (use hpo.jax.org/app)**
Myocarditis

**Comment by pnrobinson** [comment]
Please use a modifer term to capture "recurrent"
https://hpo.jax.org/app/browse/term/HP:0031796
